### PR TITLE
Minor settings and configuration updates

### PR DESCRIPTION
**Pull request by DovydasNavickas**
/cc @PinpointTownes

### CODE REVIEW

File: samples/Mvc/Mvc.Client/Properties/launchSettings.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 {
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:7731/",
```
Comment (kevinchalet) [inline samples/Mvc/Mvc.Client/Properties/launchSettings.json]: This URL doesn't match the port we expect in the OIDC configuration. Please consider fixing it.

File: samples/Mvc/Mvc.Server/Properties/launchSettings.json [hunk 2]
```diff
@@ -1,4 +1,12 @@
 {
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:7729/",
```
Comment (kevinchalet) [inline samples/Mvc/Mvc.Server/Properties/launchSettings.json]: Same remark.

File: samples/Mvc/Mvc.Client/Properties/launchSettings.json (hunk 1 above)
Comment (DovydasNavickas) [inline samples/Mvc/Mvc.Client/Properties/launchSettings.json]: Snap. Checked only SignalR sample. Looking into MVC one now..

**Comment by DovydasNavickas** [comment]
I hope now ports are correct.

File: AspNet.Security.OpenIdConnect.Samples.sln [hunk 3]
```diff
@@ -13,6 +13,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SignalR", "SignalR", "{61F3
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Mvc", "Mvc", "{139182EF-E215-4E9C-91AA-312F29B33A76}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{57AA233F-DD3B-4971-9C24-D024E07656EE}"
```
Comment (kevinchalet) [inline AspNet.Security.OpenIdConnect.Samples.sln:16]: I'm not a huge fan of these default items, but if you think they are important, feel free to keep them.

Comment (DovydasNavickas) [inline AspNet.Security.OpenIdConnect.Samples.sln:16]: I like to have quick access to global.json and NuGet.config and as we talked about my situation in PM, they do come in handy.

Comment (kevinchalet) [inline AspNet.Security.OpenIdConnect.Samples.sln:16]: Okay, I'm sold :smile:

**Comment by kevinchalet** [comment]
Merged, thanks! https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Samples/commit/5eb5417d75ab6bc7807d60beed781b427ef14b8d :clap:

**Pull request closed without merging**